### PR TITLE
Updates to Hospitals page

### PR DESCRIPTION
**Pull request by scottduanev2**
This PR addresses most of the issues remaining on the MVP of the hospitals page, namely:

- Adds a header
- Validates that fields have been filled out before the form can be submitted
- Fixes some styling stuff with the form 

There's still some stuff left to do such as adding a controller test, but this gets most of it.

### CODE REVIEW

File: app/views/cities/index.js.erb [hunk 1]
```diff
@@ -27,11 +36,12 @@ const updateCity = (city_element, city) => {
   }
 }
 
-[[volunteer_city_element, volunteer_city], [hospital_city_element, hospital_city]].forEach((pair) => {
+[[volunteer_city_element, volunteer_city, null], [hospital_city_element, hospital_city, hosp_city_container]].forEach((pair) => {
```
Comment (kopchickm) [inline app/views/cities/index.js.erb]: This'll have to be updated for the partners page, too.

File: app/assets/stylesheets/theme.scss [hunk 2]
```diff
@@ -99,7 +99,7 @@ body {
   font-size: 42pt;
 }
 
-.lives {
+.red-emphasis {
```
Comment (kopchickm) [inline app/assets/stylesheets/theme.scss]: 👍

File: app/controllers/hospitals_controller.rb [hunk 3]
```diff
@@ -5,6 +5,8 @@ def index
     @countries_collection = @countries_collection.sort_by {|c| c[0]}
 
     @states_collection = []
+
+    @can_submit = false
```
Comment (kopchickm) [inline app/controllers/hospitals_controller.rb]: Is `@can_submit` being used elsewhere? I don't see it in the PR.

**Comment by hantuzun** [comment]
I didn't see this previously. Apologies. So I should deal with merging this myself :)

**Pull request closed without merging**